app/docs/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "pnpm run start",
     "start": "vitepress dev src",
-    "build": "vitepress build src"
+    "build": "vitepress build src",
+    "generate-api-docs": "npx widdershins --omitHeader --search false --language_tabs \"typescript:TypeScript\" --summary ../../bin/openapi-extractor/data/openapi.json -o src/reference/external-api.md -u ./templates"
```

Comment:
Does the `generate-api-docs` work from any directory in the repository? I see the `../../bin/<etc>` path and it made me curious.

I'm not going to be hard on the specifics here because seriously nice work getting this into buck2. We can make changes later if need be.

Reply:
I will be following up with changes for this - I need to understand how path traversal will work better here